infra/modules/azure_function_app/variables.tf

```diff
@@ -15,7 +15,12 @@ variable "environment" {
 
   validation {
     condition     = length("${var.environment.prefix}${var.environment.env_short}reg${var.environment.domain == null ? "" : replace(var.environment.domain, "-", "")}${var.environment.app_name}stfn${var.environment.instance_number}") <= 24
-    error_message = "Storage Account name must have less than 25 characters. Current value is \"${var.environment.prefix}${var.environment.env_short}reg${var.environment.domain == null ? "" : var.environment.domain}${var.environment.app_name}st${var.environment.instance_number}\""
+    error_message = "Storage Account name must have less than 25 characters. Current value is \"${var.environment.prefix}${var.environment.env_short}reg${var.environment.domain == null ? "" : var.environment.domain}${var.environment.app_name}stfn${var.environment.instance_number}\""
```

Comment:
why the 'reg' infix ? it's difficult to obtain less than 25 chars with this naming ie.

https://github.com/pagopa/dx-playground/actions/runs/12938884631/job/36090011729#step:9:341

"dxdregplaygrounddfstfnd01"

Reply:
´reg´stands for region and is a placeholder (I can't get the real value here).

I propose to use `stfd` instead of `stfnd`as resource suffix, to match the length capacity of the host's storage account (which is `stfn`)

Reply:
this would be a breaking change for existing storages, isn't it?

Reply:
It is not as we're introducing the durable storage account in this pr